Subject: [PATCH] Update Eigen SHA256SUM

---
 deps/+Eigen/Eigen.cmake | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/deps/+Eigen/Eigen.cmake b/deps/+Eigen/Eigen.cmake
index e14b0f8fa..2b0163714 100644
--- a/deps/+Eigen/Eigen.cmake
+++ b/deps/+Eigen/Eigen.cmake
@@ -1,4 +1,4 @@
 add_cmake_project(Eigen
     URL "https://gitlab.com/libeigen/eigen/-/archive/3.3.7/eigen-3.3.7.zip"
-    URL_HASH SHA256=e09b89aae054e9778ee3f606192ee76d645eec82c402c01c648b1fe46b6b9857
+    URL_HASH SHA256=4815118c085ff1d5a21f62218a3b2ac62555e9b8d7bacd5093892398e7a92c4b
 )
-- 
2.39.5

