From f93f8c8999db55252991fd55180140eb2e474205 Mon Sep 17 00:00:00 2001
Date: Sat, 21 Sep 2024 17:25:37 -0700
Subject: [PATCH] Set GLAD_GLES2_USE_SYSTEM_EGL

---
 src/libvgcode/glad/include/glad/gles2.h | 4 ++++
 1 file changed, 4 insertions(+)

diff --git a/src/libvgcode/glad/include/glad/gles2.h b/src/libvgcode/glad/include/glad/gles2.h
index 2983e1df5..5691317c1 100644
--- a/src/libvgcode/glad/include/glad/gles2.h
+++ b/src/libvgcode/glad/include/glad/gles2.h
@@ -87,6 +87,10 @@ extern "C" {
   #endif
 #endif
 
+#ifndef GLAD_GLES2_USE_SYSTEM_EGL
+  #define GLAD_GLES2_USE_SYSTEM_EGL 1
+#endif
+
 #ifndef GLAD_PLATFORM_UWP
   #if defined(_MSC_VER) && !defined(GLAD_INTERNAL_HAVE_WINAPIFAMILY)
     #ifdef __has_include
-- 
2.46.0

