From b28998b243239b8d1e2e0a8d370bb336c7faf1ab Mon Sep 17 00:00:00 2001
Date: Wed, 26 Mar 2025 09:02:31 +0000
Subject: [PATCH] Use alternate location for GMP

---
 deps/+GMP/GMP.cmake | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/deps/+GMP/GMP.cmake b/deps/+GMP/GMP.cmake
index 724db2eea..c3e88173f 100644
--- a/deps/+GMP/GMP.cmake
+++ b/deps/+GMP/GMP.cmake
@@ -54,7 +54,7 @@ else ()
 
     ExternalProject_Add(dep_GMP
         EXCLUDE_FROM_ALL ON
-        URL https://gmplib.org/download/gmp/gmp-6.2.1.tar.bz2
+        URL https://ftp.gnu.org/gnu/gmp/gmp-6.2.1.tar.bz2
         URL_HASH SHA256=eae9326beb4158c386e39a356818031bd28f3124cf915f8c5b1dc4c7a36b4d7c
         DOWNLOAD_DIR ${${PROJECT_NAME}_DEP_DOWNLOAD_DIR}/GMP
         BUILD_IN_SOURCE ON 
-- 
2.39.5

